### DISCUSSION TITLE
Dart_PostCObject_DL How to send multiple data to dart?

### DISCUSSION THREAD

**Discussion by LEggcookies** [dart-lang/ffi#211] [Q&A] [answered]
![image](https://github.com/dart-lang/ffi/assets/33369299/e0da4aa9-e406-40ec-895f-cdad302a8067)
I want to send all the data to dart. The above are some of my attempts, but they all failed. Please tell me how to do it. 
Thanks!

**Comment by dcharkes** [comment, marked as answer]
Here's an example: https://github.com/dart-lang/sdk/blob/4bddf8a25ff894f6a8ac9ee122a1eb2112fd2dd1/runtime/bin/ffi_test/ffi_test_functions_vmspecific.cc#L781

`Dart_CObject_kArray` is the right approach, see the example on how to use it.
